lms/validation/authentication/_exceptions.py

```diff
@@ -4,27 +4,12 @@
 # ValidationError has a large hierarchy, but we need to inherit from it
 
 __all__ = [
-    "ExpiredSessionTokenError",
-    "MissingSessionTokenError",
-    "InvalidSessionTokenError",
     "MissingStateParamError",
     "ExpiredStateParamError",
     "InvalidStateParamError",
 ]
 
 
-class ExpiredSessionTokenError(ValidationError):
```

Comment:
I don't think these types add much information. They are all `ValidationError` and just instantiated and raised once. 

Just raising `ValidationError` seems more straightforward and a new type can be created when different behavior is needed.

Reply:
Yep. This all seems very roundabout.